### PR TITLE
Fix logging

### PR DESCRIPTION
**Pull request by StefanCholakov**
**Description**

Changes proposed in this pull request:

- Improve error logging

### CODE REVIEW

**Review by StanislavStefanov — APPROVED** [review]

**Comment by ataleksandrov** [comment]
Don't forget to bump the components versions.

**Review by StanislavStefanov — DISMISSED** [review]

**Review by StanislavStefanov — DISMISSED** [review]

File: components/tenant-fetcher/internal/tenant/service.go [hunk 1]
```diff
@@ -51,14 +51,14 @@ func (s *service) Create(writer http.ResponseWriter, request *http.Request) {
...
-		logger.WithError(err).Error("while extracting request body")
+		logger.WithError(err).Errorf("while extracting request body: %v", err)
 		http.Error(writer, err.Error(), http.StatusInternalServerError)
 		return
 	}
 
 	tenantId := gjson.GetBytes(body, s.config.TenantProviderTenantIdProperty)
 	if !tenantId.Exists() || tenantId.Type != gjson.String || len(tenantId.String()) == 0 {
-		logger.Errorf("Property %q not found in body or it is not of String type", s.config.TenantProviderTenantIdProperty)
+		logger.Errorf("Property %q not found in body or it is not of String type: %v", s.config.TenantProviderTenantIdProperty, err)
```
Comment (desislavaa) [inline components/tenant-fetcher/internal/tenant/service.go:61]: no error here

File: components/tenant-fetcher/internal/tenant/service.go [hunk 2]
```diff
@@ -110,21 +110,21 @@ func (s *service) DeleteByExternalID(writer http.ResponseWriter, request *http.R
...
```
Comment (desislavaa) [inline components/tenant-fetcher/internal/tenant/service.go:120]: no error here

**Review by desislavaa — APPROVED** [review]

**Pull request merged**